ui_buttons.h

```diff
@@ -12,6 +12,7 @@ static void button_setcolors_success(BUTTON *b)
     b->c3 = COLOR_BUTTON_SUCCESS_HOVER_BACKGROUND;
     b->ct1 = COLOR_BUTTON_SUCCESS_TEXT;
     b->ct2 = COLOR_BUTTON_SUCCESS_HOVER_TEXT;
+    b->disabled = 0;
```

Comment:
Can you think of a better way that doesn't duplicate code too much? I could put `b->disabled = 1`/`b->disabled = 0` outside those functions, but that would only add more unneeded code. Or should I just rename color functions?